simple-mbed-cloud-client.cpp

```diff
@@ -86,6 +86,9 @@ int SimpleMbedCloudClient::init() {
         return 1;
     }
 
+    //Initialise the storage
+    int mount_result = mount_storage();
```

Comment:
No handling for the ret-code.

Reply:
Is there effects when mounting twice?

Reply:
@dhwalters423 https://github.com/ARMmbed/mbed-os/blob/a3d8c8fe161e27a99876ab9db1bd1adf6a20e49a/features/filesystem/fat/FATFileSystem.cpp#L295

It looks like `mount` should return `-EINVAL` if it is mounted twice.

Reply:
I'll go with @bridadan suggestion and catch the error of unmounted fs and then mount if needed..